### ISSUE TITLE
Broken in WordPress 5.4

### ISSUE THREAD

**Issue by thefrosty** [humanmade/Restsplain#26]
Assuming a new version of React is bundled with WordPress 5.4, which broke Restplain. Testing this on a site in 5.3 and 5.4 to verify.

Console errors:
<img width="1734" alt="Screen Shot 2020-04-04 at 6 09 04 PM" src="https://user-images.githubusercontent.com/367897/78464520-6d121280-769f-11ea-9627-f33a4adbdfb0.png">

**Comment by mateuswetah** [comment]
I can confirm this bug, I had to downgrade for now :/

**Comment by roborourke** [comment]
Yeah it's a super old version of React. Looks like issue #19 would solve this.
